engine/src/main/java/com/datatorrent/stram/StreamingContainerManager.java

```diff
@@ -230,7 +231,16 @@ protected boolean removeEldestEntry(Map.Entry<String, ContainerInfo> eldest)
   public static class CriticalPathInfo
```

Comment:
@vrozov The class is actually exposed outside of StreamingContainerManager via a get method.